tests/functional/kata-agent-apis/setup_common.sh

```diff
@@ -32,6 +32,9 @@ agent_log_file="${PWD}/kata-agent.log"
 agent_log_level="debug"
 keep_logs=false
 
+local_policy_file="/opt/kata/test.rego"
+policy_file="/etc/kata-opa/default-policy.rego"
```

Comment:
@Sumynwa , using these paths on a test machine that might be used for something else too makes me nervous. Would it be practical to add an Agent command line parameter that overrides the default path to the Policy? If not, at least a new agent/src/config.rs field to configure the same type of Policy path?